docs/upgrading_to_v36.0.md

```diff
@@ -25,6 +25,91 @@ The variable `logging_variant` has been removed from the Autopilot sub-modules.
 }
 ```
 
+### ACM Sub-Module Removal
+The ACM Sub-Module has been removed in v36.0.  Please use the [google_gke_hub_feature](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/gke_hub_feature#example-usage---enable-fleet-default-member-config-configmanagement) and [google_gke_hub_feature_membership](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/gke_hub_feature_membership#example-usage---config-management-with-git) resources.  For additional examples, see [terraform-docs-samples/gke/autopilot/config_sync](https://github.com/terraform-google-modules/terraform-docs-samples/tree/main/gke/autopilot/config_sync) and [terraform-docs-samples/gke/autopilot/policycontroller](https://github.com/terraform-google-modules/terraform-docs-samples/tree/main/gke/autopilot/policycontroller).
+
+
+```diff
+-module "acm" {
+-  source  = "terraform-google-modules/kubernetes-engine/google//modules/acm"
+-  version = "~> 35.0"
+-  project_id   = var.project_id
+-  location     = module.gke.location
+-  cluster_name = module.gke.name
+
+-  # Config Sync
+-  enable_config_sync = true
+-  sync_repo   = "git@github.com:GoogleCloudPlatform/anthos-config-management-samples.git"
+-  sync_branch = "1.0.0"
+-  policy_dir  = "foo-corp"
+
+-  # Policy Controller
+-  enable_policy_controller = true
+-  enable_fleet_feature = true
+-  install_template_library = true
+-  enable_referential_rules = true
+-  policy_bundles = ["https://github.com/GoogleCloudPlatform/acm-policy-controller-library.git/bundles/pss-baseline-v2022"]
+-}
+
+# Config Sync
++resource "google_gke_hub_feature" "config_feature" {
++  project  = var.project_id
++  location = "global"
++  name     = "configmanagement"
++}
+
++resource "google_gke_hub_feature_membership" "config_feature_membership" {
++  project  = var.project_id
++  location = "global"
+
++  feature             = google_gke_hub_feature.config_feature.name
++  membership          = module.gke.fleet_membership
++  membership_location = module.gke.region
+
++  configmanagement {
++    config_sync {
++      enabled = true
++      git {
++        sync_repo = "git@github.com:GoogleCloudPlatform/anthos-config-management-samples.git"
++        sync_branch = "1.0.0"
```

Comment:
nit: https://github.com/GoogleCloudPlatform/anthos-config-management-samples/ doesn't have 1.0.0 branch anymore. Should we use a branch that actually exists, such as `main`?

Reply:
Thanks @janetkuo! Sure, in theory this shouldn't be used as-is, but we can certainly switch to `main` which I see if the default branch.